cli/parse_pdfs.py

```diff
@@ -249,7 +261,7 @@ def parse_file(
         },
     )
 
-    output_path = output_dir / f"{input_task.document_id}.json"
+    output_path = cast(Path, output_dir / f"{input_task.document_id}.json")
```

Comment:
What's the need for this line, wondering if it will affect S3Paths? 
Have we tested with --s3 if it is needed.

Reply:
You're right. This isn't needed. This is related to the error we were discussing on Monday with the logging. Will edit.